flink-staging/flink-streaming/flink-streaming-connectors/pom.xml

```diff
@@ -46,9 +53,13 @@ under the License.
 		<dependency>
 			<groupId>org.apache.kafka</groupId>
 			<artifactId>kafka_2.10</artifactId>
-			<version>0.8.2.0</version>
+			<version>${kafka.version}</version>
 			<exclusions>
```

Comment:
If this is only about the different names of `leader.getConnectionString()`, we can also play some reflection tricks.